services/app-api/src/resolvers/healthPlanPackage/fetchHealthPlanPackage.ts

```diff
@@ -58,7 +59,11 @@ export function fetchHealthPlanPackageResolver(
                     'user not authorized to fetch data from a different state'
                 )
             }
-        } else if (isCMSUser(context.user) || isAdminUser(context.user)) {
+        } else if (
+            isCMSUser(context.user) ||
+            isAdminUser(context.user) ||
+            isHelpdeskUser(context.user)
```

Comment:
With this new type of user who cannot fetch a draft, we should change the error message `'CMS user not authorized to fetch a draft'` to something more generic.

Reply:
Unless help desk users are considered CMS users?